test/gtest/uct/test_p2p_am.cc

```diff
@@ -33,6 +33,11 @@ class uct_p2p_am_test : public uct_p2p_test
         uct_p2p_test(sizeof(receive_desc_t)), m_am_count(0), m_am_posted(0),
         m_keep_data(false)
     {
+        m_pending_req.sendbuf = NULL;
+        m_pending_req.test = this;
```

Comment:
it adds double initialization in `am_handler_resp` which may be removed.

Reply:
Thanks for clarification WRT POD!
minor: lets init `test` with  `NULL` here as all other non-initialized fields and fill valid data in single place.